tests/behat/behat_mod_coursework.php

```diff
@@ -2914,13 +2914,15 @@ protected function create_user($rolename, $displayname = '') {
         $generator = testing_util::get_data_generator();
 
         $user = new stdClass();
+        // Use a random string in password to stop Chrome complaining that password exposed in data breach.
```

Comment:
Unrelated change - needs dropping

This makes it far harder to debug behat tests and is not necessary if you're using behat correctly.
See https://developer.chrome.com/blog/chrome-for-testing